RELEASE

```diff
@@ -0,0 +1,8 @@
+tag: v0.0.0
```

Comment:
I created and pushed that tag, on commit `412b849 - Trace viewer prototype application` (the second commit in this repo, right after "Initial commit"). This way, the first generated release notes will go back to the start of the repo's history (we can edit them after as needed)

Reply:
There is already a 0.1.0 release that was manually pushed to npm and a tag in the repo. Shouldn't you base the new release on the 0.1.0 tag instead?

Reply:
> We could, but then the auto-generated release notes would start at v0.1.0. The plan is, after the next release, I will take the release notes and move those relevant to v0.1.0 in a manually-created GitHub release (from that tag). I did similar for time-line chart: https://github.com/eclipse-cdt-cloud/timeline-chart/releases

Reply:
timeline-chart: original release notes / GH release, that I manually massaged after release:
https://github.com/eclipse-cdt-cloud/timeline-chart/pull/274#issuecomment-1973707287

Reply:
Right. Forgot about it. Makes sense.

Reply:
BTW, I'm not sure I was totally clear - the discussed tag (v0.0.0) will only affect the starting point for the auto-generated release notes, when we proceed with the first release, in another PR. This PR here will not result in a release, just setting the stage.

Reply:
thanks for clarifying